packages/angular_devkit/schematics/src/tree/action.ts

```diff
@@ -157,13 +157,15 @@ export function isContentAction(action: Action): action is CreateFileAction | Ov
 }
 
 
-export function isAction(action: any): action is Action {  // tslint:disable-line:no-any
-  const kind = action && action.kind;
+export function isAction(action: unknown): action is Action {
```

Comment:
This is a breaking changing, because `unknown` is more restrictive than `any`.

Reply:
I don't think this is breaking. `unknown` is a top-type like `any`, such that anything is assignable to it. It's only within this fn's implementation that more restrictions apply when using `action` (although that's immediately undone with the speculative cast to `Action`)

Reply:
You are absolutely right, probably in this case it’s best to remove the cast and check add a check using typeof

`typeof action === ‘object’ && action.kind`

Reply:
Hi @alan-agius4, let me know if any changes are needed.

Reply:
Nit: can you use === instead of ==?